src/tests/subscriptions.test.js

```diff
@@ -57,9 +57,9 @@ describe('GET /api/subscription/:id', () => {
       .get(`/api/subscription/${mockSubId}`);
     expect(response.status).toBe(200);
     expect(response.error).toBeFalsy();
-    expect(response.body.data.classes).toBe(subsSeed[0].classes);
-    expect(response.body.data.members).toBe(subsSeed[0].members);
-    expect(response.body.data.date).toBe(subsSeed[0].date);
```

Comment:
this tests didn't passed? why?

Reply:
because they didn't check the populated classes and members, they were just checking the ID